discord/appinfo.py

```diff
@@ -52,9 +55,28 @@ class AppInfo:
         grant flow to join.
     rpc_origins: Optional[List[:class:`str`]]
         A list of RPC origin URLs, if RPC is enabled.
+    summary: Optional[:class:`str`]
```

Comment:
```suggestion
    summary: :class:`str`
```

This seems to always be there as a string.